static/app/utils/issueTypeConfig/replayConfig.tsx

```diff
@@ -5,9 +5,18 @@ const replayConfig: IssueCategoryConfigMapping = {
   _categoryDefaults: {
     actions: {
       archiveUntilOccurrence: {enabled: true},
-      delete: {enabled: false},
-      deleteAndDiscard: {enabled: false},
-      merge: {enabled: false},
+      delete: {
+        enabled: false,
+        disabledReason: t('Not yet supported for replay issues'),
+      },
+      deleteAndDiscard: {
+        enabled: false,
+        disabledReason: t('Not yet supported for replay issues'),
+      },
+      merge: {
+        enabled: false,
+        disabledReason: t('Not yet supported for replay issues'),
```

Comment:
I would change these to straight up "not supported for replay issues" - don't think either of these really make sense?

Reply:
ya, merge probably never supported... both `delete` as well as `delete and discard` could be supported tho right?